app/components/root/TeacherListSearch.tsx

```
@@ -0,0 +1,72 @@
+            postNewMatchingAcceptance(
+              {
+                matchingId,
+                userIds: selectedTeachers,
+              },
+              {
+                onSuccess: (data) => {
+                  alert(data.data);
+                  closeModal();
+                },
+                onError: (error) => {
+                  const errorMessage = "알림톡 발송 중 문제가 발생했습니다. 잠시 후 다시 시도해주세요.";
```

ESLint

[error] Insert `⏎···················`

(prettier/prettier)